src/peft/tuners/ia3.py

```diff
@@ -303,6 +303,8 @@ def _set_adapter_layers(self, enabled=True):
         for module in self.model.modules():
             if isinstance(module, IA3Layer):
                 module.disable_adapters = False if enabled else True
+            elif isinstance(module, ModulesToSaveWrapper):
+                module.disable_adapter = False if enabled else True
```

Comment:
Thanks for taking a look.

Good point about `disable_adapter`. I was going with singular since it's only a single adapter, but I can see your argument too. Let's wait what the other reviewers prefer.

Reply:
Yeah, let's have `disable_adapters` for homogeneity.

Reply:
Done